### DISCUSSION TITLE
Parcels run time and animation

### DISCUSSION THREAD

**Discussion by ese-jlk18** [Parcels-code/Parcels#1001] [Q&A]
I am currently plotting trajectories for a couple hundred of JITparticles on a U,V vector field from a netCDF file see attached for python code. I am wondering if it is normal for the code to take unto a day to run and if not if someone could highlight my errors?

Secondly is there a way to prevent a graphical output of the animation at every time step?

Thank you in advance 
<img width="1081" alt="Screen Shot 2021-02-23 at 1 26 32 PM" src="https://user-images.githubusercontent.com/57348151/108855217-acf82600-75e0-11eb-955e-421cd2e54dfa.png">

**Comment by reint-fischer** [comment]
Hi @ese-jlk18,

Depending on the size of your netCDF and the location of your particles, the code may run for a day, but from what I can see with your timesteps, that should not need to be the case. If you want to prevent the graphical output from showing, you can remove the `moviedt` argument from your `pset.execute` statement. This could also improve your simulation speed.

Another note is that usually in the definition of the FieldSet, the 'filenames' dictionary is passed, instead of the single netCDF filename, although if the filename is the same for U and V this should not be a problem.

I hope this helps
